docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,45 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+Clone the main REANA repository:
+
+```console
+$ git clone https://github.com/reanahub/reana.git && cd reana
+```
+
+You can setup a REANA development as follows:
+```console
+$ GITHUB_USER=<YOUR-GITHUB-USERNAME> make setup clone prefetch
+```
+
+
+Deploy REANA in development mode with code reload and [debugging capabilities](debugging):
+
+```console
+$ CLUSTER_FLAGS=debug.enabled=true make build deploy
```

Comment:
- `CLUSTER_FLAGS="debug.enabled=true" ` would be more readable WRT double equal sign